windows-app-samples/README.md

```diff
@@ -0,0 +1,15 @@
+# WindowsAppSamples
+
+Windows apps can be scanned for accessibility issues using the tools in the [Axe.Windows](https://github.com/microsoft/axe-windows) repo. These tools do not currently produce Sarif files. The general pattern is as follows:
```

Comment:
We've seen some folks infer that "app" refers specifically to a "store app"; spelling out "application" might make it a little more clear that it's not specific to that

Reply:
Done